NOTICE

```diff
@@ -0,0 +1,15 @@
+   Red Hat 3scale API Management Operations
+   Copyright (c) 2010-2016 3scale, Inc
+   Copyright (c) 2016-2018 Red Hat, Inc.
```

Comment:
Maybe just use the latter line and 2019 as the year?

Reply:
sure!

Reply:
solved on 449bbdc04398a672e836e3e2970e735dc936d2b4